doctr/models/recognition/sar/tensorflow.py

```diff
@@ -150,7 +149,8 @@ def call(
             # shape (N, rnn_units + 1) -> (N, vocab_size + 1)
             logits = self.output_dense(logits, **kwargs)
             # update symbol with predicted logits for t+1 step
-            if kwargs.get('training'):
+            if kwargs.get('training') and gt is not None:
```

Comment:
can be reverted since we already ensure that the gt is provided in training mode above